### ISSUE TITLE
add warning when representative with too much voting power is used

### ISSUE THREAD

**Issue by ThunderFD** [getcanoe/canoe#181]
if an address in the wallet has a rep with more than X% voting power, a warning should pop up on the main screen above the accounts, advising the user to change their rep for "account_name"

![image](https://user-images.githubusercontent.com/17814079/37155240-acb8d62a-22e2-11e8-897c-f1c037a7e132.png)
_(shamelessly grabbed from google images)_

maybe with a "tap here" link/button that leads the user to a page with more instructions.

**Comment by stefonarch** [comment]
This leads to the discussion "average users who don't care, aren't interested in and will do nothing but are scared" vs   informed and interested people.
It could be shown like the "rate Canoe" window, once every 5 days.

**Comment by ThunderFD** [comment]
yeah, it's definitely up to debate how exactly this should look like, but I think the underlying idea is great to push decentralization.

**Comment by rtaibah** [comment]
Well for starters we can do it in a less intimidating color. Have a
friendly treatment of it, and explain what is value of having reps more
spread up.

**Comment by ceddup** [comment]
I'd do a message that doesn't scare people at all (techy or not) but that is bugging until you do it. 
After all, everything will end up to be free for all (Nano network, Canoe) and at the end of the day, it will be the only 'work' asked to people; for their own safety.
I think any Nano user will have to get that concept eventually. I would have never believed that my granma would ever use internet back in the 90s. Do you see my point ?

**Comment by stefonarch** [comment]
Not sure about that... 95% of windows user never changed the default wallpaper, fear to make damage...

**Comment by ceddup** [comment]
Actually I agree with you no damage shall be done. 
This warning could start to show from times to times after some while. The point is, if people were told that their windows box (and all of windows boxes) could get hacked if they don't change the default wallpaper, they sure would do it. 
In a world where windows would be free as beer...

**Comment by stefonarch** [comment]
There is this thing popping in every X days, asking for rating, so we have it.

**Comment by ceddup** [comment]
I never saw that thing ;-)

**Comment by stefonarch** [comment]
Me twice until now, it allows to select 1-5 stars , it's in POEdit too.

**Comment by ThunderFD** [comment]
haven't seen that so far either..
imo the rep thing should pop up right when it notices the issue.
the user could X it away and then it could show up after 2 weeks or a month again or so.
or the user could just leave it there to look at it later.
